crates/bevy_scene/src/serde.rs

```diff
@@ -14,19 +16,28 @@ use serde::{
 };
 use std::fmt::Formatter;
 
+/// Name of the serialized scene struct type.
 pub const SCENE_STRUCT: &str = "Scene";
+/// Name of the serialized resources field in a scene struct.
 pub const SCENE_RESOURCES: &str = "resources";
+/// Name of the serialized entities field in a scene struct.
 pub const SCENE_ENTITIES: &str = "entities";
 
+/// Name of the serialized entity struct type.
 pub const ENTITY_STRUCT: &str = "Entity";
+/// Name of the serialized component field in an entity struct.
 pub const ENTITY_FIELD_COMPONENTS: &str = "components";
 
+/// Handles serialization of a scene as a struct containing its entities and resources.
```

Comment:
Could a paragraph on usage or an example be added? It's a bit tricky to use, so justified.